src/components/forms/formr-part-a/ProfileToFormRPartAInitialValuesMapping.ts

```diff
@@ -1,15 +1,24 @@
 import { TraineeProfile } from "../../../models/TraineeProfile";
 import { FormRPartA } from "../../../models/FormRPartA";
 
-export function mapProfileToFormRPartAInitialValues(
+export function ProfileToFormRPartAInitialValuesMapping(
   traineeProfile: TraineeProfile | null
 ): FormRPartA | null {
   if (!traineeProfile) {
     return null;
   }
 
   const pd = traineeProfile.personalDetails;
-  const programme = traineeProfile.programmeMemberships[0];
+  const programme = traineeProfile.programmeMemberships
+    .filter(pm => new Date(pm.startDate) < new Date())
+    .reduce(function(a, b) {
+      return a.startDate > b.startDate ? a : b;
+    });
+
+  const programmeSpecialty = programme.curricula.find(
+    c => c.curriculumSubType === "MEDICAL_CURRICULUM"
```

Comment:
If there are more than 1 curricula with "MEDICAL_CURRICULUM" subtype, we may need to find the most recent as well

Reply:
I agree with Doris, may need to apply this filter first.

Reply:
@DorisWongNHS  / @hihilary  how do we get the most recent curriculum?

Reply:
we might have to add 'curriculumStartDate' property for curriculum.